ygot/pathstrings_test.go

```diff
@@ -109,6 +110,22 @@ func TestPathToString(t *testing.T) {
 	}
 }
 
+func TestPathToStrings(t *testing.T) {
+	in := &gnmipb.Path{Elem: []*gnmipb.PathElem{
```

Comment:
Can you make this the standard `tests := []struct{ ... }; for _, tt := range tests { ... }` layout please? It just makes things easier to extend when/if there are issues.

Reply:
I usually insist on the opposite in reviews: only add extra abstraction when it's actually needed.
Given the code sharing between PathToStrings and PathToString, more extra cases can be to TestPathToString.